tests/cloud_functions/conftest.py

```
@@ -0,0 +1,111 @@
+
+@pytest.fixture(scope="session")
+def firebase_emulator():
+    """Setup Firebase emulator connection."""
+    os.environ["FIRESTORE_EMULATOR_HOST"] = "firebase-emulator:8080"
+    os.environ["GCLOUD_PROJECT"] = "test-project"
+
+    # Clean up any existing apps
+    try:
+        delete_app(get_app())
+    except ValueError:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError)`

(SIM105)